app/jobs/resolution_proofing_job.rb

```diff
@@ -37,9 +37,12 @@ def perform(
       symbolize_names: true,
     )
 
-    applicant_pii = decrypted_args[:applicant_pii]
-
     user = User.find_by(id: user_id)
+    current_sp = ServiceProvider.find_by(issuer: service_provider_issuer)
+
+    applicant_pii = decrypted_args[:applicant_pii]
+    applicant_pii[:uuid_prefix] = current_sp&.app_id
+    applicant_pii[:uuid] = user&.uuid
```

Comment:
it would be very bad if there was no user found, right? like this *should* error on a nil user?
```suggestion
    applicant_pii[:uuid] = user.uuid
```

Reply:
Agreed, probably want a test for that case as well

Reply:
Yeah, I think that is right. I believe the only way we get in this scenario is if a user immediately deletes their account upon submitting a job. And we probably do want to error in that case. I was copying a pattern I saw elsewhere in the job

https://github.com/18F/identity-idp/blob/7558431d47d3c951ed0b29278c0b82dc631d9da9/app/jobs/resolution_proofing_job.rb#L82

https://github.com/18F/identity-idp/blob/7558431d47d3c951ed0b29278c0b82dc631d9da9/app/jobs/resolution_proofing_job.rb#L105

If we are going to error here if the user is absent I'll clean those spots up as well.